docs/language/expressions.rst

```diff
@@ -227,7 +228,8 @@ Casting
 _______
 
 Solidity is very strict about the sign of operations, and whether an assignment can truncate a
-value. You can force the compiler to accept truncations or differences in sign by adding a cast.
+value. You can force the compiler to accept truncations or differences in sign by adding an
```

Comment:
```suggestion
value. You can force the compiler to accept truncations or sign shifts by adding an
```
`Differences in sign` sounds awkward to me.

Reply:
I don't think `sign shifts` is very clear. I've changed it to `sign changes`